deploy/relay-deploy.sh

```diff
@@ -39,12 +34,15 @@ install_docker() {
 }
 
 make_relay_service() {
+    mkdir /opt/workspace
+    cd /opt/workspace
+    git clone https://github.com/OpenDDS/opendds-smart-lock.git
```

Comment:
Instead of using `git clone` here, this script (relay-deploy.sh) could take the ini file as a command line argument.